examples/mcu-board-support/stm32u5g9j_dk2.rs

```diff
@@ -0,0 +1,426 @@
+                    drawn = true;
+                });
+
+                if drawn {
+                    scb.clean_dcache_by_slice(work_fb);
+                    // Safety: the frame buffer has the right size
+                    ltdc.set_buffer(LtdcLayer::Layer1, work_fb.as_ptr() as *const ())
+                        .await
+                        .unwrap();
+                    // Swap the buffer pointer so we will work now on the second buffer
+                    core::mem::swap::<&mut [_]>(&mut work_fb, &mut displayed_fb);
+                }
+
+                match touch.get_touch(&mut i2c) {
+                    Ok(point) => {
+                        let button = PointerEventButton::Left;
+                        let event = match point {
+                            Some(point) => {
+                                let position =
+                                    slint::PhysicalPosition::new(point.x as i32, point.y as i32)
+                                        .to_logical(window.scale_factor());
+                                Some(match last_touch.replace(position) {
+                                    Some(_) => WindowEvent::PointerMoved { position },
+                                    None => WindowEvent::PointerPressed { position, button },
+                                })
+                            }
+                            None => last_touch
+                                .take()
+                                .map(|position| WindowEvent::PointerReleased { position, button }),
+                        };
+
+                        if let Some(event) = event {
+                            let is_pointer_release_event =
+                                matches!(event, WindowEvent::PointerReleased { .. });
+                            window.dispatch_event(event);
+
+                            // removes hover state on widgets
+                            if is_pointer_release_event {
+                                window.dispatch_event(WindowEvent::PointerExited);
+                            }
+                        }
+                    }
+                    Err(gt911::Error::I2C(e)) => {
+                        defmt::error!("failed to get touch point: {:?}", e);
+                    }
+                    Err(_) => {
+                        // ignore as these are expected NotReady messages from the touchscreen
+                    }
+                }
+            }
+
+            /*
+            Use this when the async gt911 API works:
+
+            if let Some(duration) = slint::platform::duration_until_next_timer_update()
+                .and_then(|core_duration| embassy_time::Duration::try_from(core_duration).ok())
+            {
+                embassy_time::Timer::after(duration).await;
+            }
+            */
+        }
+    }
+}
+
+#[embassy_executor::task()]
+async fn main_loop_task(backend: &'static StmBackend) {
+    backend.run_loop().await;
+}
+
+struct DummyLoopProxy;
```

Comment:
you can return None from new_event_loop proxy instead.

Reply:
Good idea, will do that.